.github/workflows/linux-docker-packaging-appimage.yml

```diff
@@ -0,0 +1,40 @@
+name: Docker for Appimage packaging
+on:
+  push:
+    branches:
+      - master
+      - main
+      - dev
+    tags:
+      - '*'
+
+env:
+  DOCKER_IMAGE_NAME: ghcr.io/noxworld-dev/build-docker
```

Comment:
Better to cange the image name - eg to:
`ghcr.io/noxworld-dev/appimage-docker`
Don't worry, the package names are not tied to repo names, they are tied only to the workspace (`noxworld-dev` part).